.github/workflows/.deploy.yml

```diff
@@ -137,8 +137,10 @@ jobs:
           oc_server: ${{ vars.OC_SERVER }}
           oc_token: ${{ secrets.OC_TOKEN }}
           overwrite: ${{ matrix.overwrite }}
+
           parameters:
-            -p ZONE=${{ inputs.target }} -p TAG=${{ needs.init.outputs.tag }}
-            -p URL=${{ needs.init.outputs.url }} -p REPLICA_COUNT=1
```

Comment:
REPLICA_COUNT=1 is removed. Just use the default value from individual component openshift.deployment.yml. The db component does not have this parameter being used (so it is 1).